### PR TITLE
Cocoapods support

### PR DESCRIPTION
**Pull request by Sajjon**
I have already pod trunk pushed the podspec. But the `s.source` should be changed to this repo and to be using a tag, I set the version to a future tag `0.3.3`.

So to fix proper cocoapod support you need to:
* Inform me of your email so I can add you as owner of Pod, more info: https://guides.cocoapods.org/making/getting-setup-with-trunk.html#adding-other-people-as-contributors
* Accept this PR
* Change the `s.source` in podspec to `{ :git => 'https://github.com/hyugit/EllipticCurve.git', :tag => s.version }`
* `git tag '0.3.3' && git push --tags`
* After I have added you as owner: `pod trunk push`

Thanks for a great project!

### CODE REVIEW

File: EllipticCurve.podspec [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  s.version      = '0.3.2'
+  s.summary      = 'ECDSA in pure Swift'
+
+  s.description  = <<-DESC
+                  Elliptic Curve Digital Signing Algorithms in pure Swift
+                   DESC
+
+  s.homepage     = 'https://github.com/hyugit/EllipticCurve.git'
+  s.license      = 'MIT'
+  s.authors      = { 'Huang Yu' => '13056774+mryu87@users.noreply.github.com', 'Alexander Cyon' => 'alex.cyon@gmail.com' }
```
Comment (hyugit) [inline EllipticCurve.podspec]: can you change my email to pod@trave7er.com? thanks.

Comment (hyugit) [reply EllipticCurve.podspec]: there are some other places that have broken references...

File: EllipticCurve.podspec [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  s.source = { :git => 'https://github.com/Sajjon/EllipticCurve.git', :branch => 'cocoapods' }
```
Comment (hyugit) [inline EllipticCurve.podspec]: also here, if you can change it to 'https://github.com/hyugit/EllipticCurve.git', it should work.

**Comment by hyugit** [comment]
Hi @Sajjon 

Are you still working on this? Shall I fix these minor issues myself?

Thanks,

Huang

**Comment by Sajjon** [comment]
@hyugit sorry for the late response I was very busy last week. I updated the podspec.

**Comment by hyugit** [comment]
no problem! thanks for the update!

**Comment by hyugit** [comment]
btw, there may be other places with broken references due to my github username changes. I'll clean it up afterwards if necessary.

**Pull request merged**